packages/bsky/src/api/app/bsky/feed/getActorLikes.ts

```diff
@@ -70,9 +71,10 @@ const skeleton = async (
   let feedItemsQb = feedService
     .selectFeedItemQb()
     .innerJoin('like', 'like.subject', 'feed_item.uri')
+    .select('like.indexedAt as sortAt')
```

Comment:
ah all the indexed record tables in the appview actually already have a sortAt column!

https://github.com/bluesky-social/atproto/blob/main/packages/bsky/src/db/tables/like.ts

Reply:
Smh ooooh I see